core/src/sigverifier/bls_sigverifier.rs

```diff
@@ -4,13 +4,27 @@
 use {
     crate::sigverify_stage::{SigVerifier, SigVerifyServiceError},
     alpenglow_vote::bls_message::BLSMessage,
-    crossbeam_channel::Sender,
+    crossbeam_channel::{Sender, TrySendError},
     solana_streamer::packet::PacketBatch,
+    std::time::{Duration, Instant},
 };
 
-#[derive(Default, Clone)]
+const STATS_INTERVAL_SECONDS: u64 = 10; // Log stats every 10 second
+
+// We are adding our own stats because we do BLS decoding in batch verification,
+// and we send one BLS message at a time. So it makes sense to have finer-grained stats
+#[derive(Debug, Default, Clone)]
+pub(crate) struct BLSSigVerifierStats {
+    pub bls_messages_sent: u64,
+    pub bls_messages_malformed: u64,
+    pub sent_failed: u64,
+    pub packets_received: u64,
```

Comment:
I think we could rename these to match the reported metrics values you used - the bls_messages prefix seems obviated by the Struct name:
```
    pub sent: u64,
    pub received_malformed: u64,
    pub sent_failed: u64,
    pub received: u64,
```

Reply:
Done.